packages/plugin/package.json

```diff
@@ -5,7 +5,7 @@
     "main": "./build/index.js",
     "license": "BSD-3-Clause",
     "dependencies": {
-        "@dhis2/analytics": "2.4.9",
+        "@dhis2/analytics": "2.4.12",
```

Comment:
Should this be part of a new version release of the plugin?

Reply:
or can we start using tilde patch `~` as per [my comment here](https://github.com/dhis2/dashboards-app/pull/822#discussion_r424260469)?

Reply:
I think adding `~` would be the right thing to do :+1: